### PR TITLE
[TG PORT] Replaces rust wave with entropic plume

### PR DESCRIPTION
**Pull request by itsmeow**
## About The Pull Request

Ports https://github.com/tgstation/tgstation/pull/52287

## Why It's Good For The Game

Rust wave is supposedly bad, but the main reason I'm porting this is so we can finally overhaul heretics. This is part of the path.

https://hackmd.io/@Khryson/BkcmBOR1P

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

image
image

</details>

## Changelog
:cl:
add: Adds entropic plume spell to heretics
del: Removes rust wave spell from heretics
fix: The ash man and rust walker nodes now link to the correct nodes.
fix: Rust's rune visual effect does not appear on reinforced walls.
/:cl:

### CODE REVIEW

**Comment by itsmeow** [comment]
Bumpy

File: code/datums/status_effects/debuffs.dm [hunk 1]
```diff
@@ -986,10 +986,69 @@
...
+/datum/status_effect/amok
+	id = "amok"
+	status_type = STATUS_EFFECT_REPLACE
+	alert_type = null
+	duration = 10 SECONDS
+	tick_interval = 1 SECONDS
+
+/datum/status_effect/amok/on_apply(mob/living/afflicted)
+	. = ..()
+	to_chat(owner, "<span class='boldwarning'>Your feel filled with a rage that is not your own!</span>")
```
Comment (Penwin0) [inline code/datums/status_effects/debuffs.dm]:
```suggestion
	to_chat(owner, "<span class='boldwarning'>You feel filled with a rage that is not your own!</span>")
```

**Review by Penwin0 — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]
Apply wiki changes and ping for merge

**Comment by itsmeow** [comment]
Wiki changes done @PowerfulBacon

**Pull request merged**